resources.json

```diff
@@ -0,0 +1,5 @@
+[
+  "https://raw.githubusercontent.com/custom-components/sensor.owlintuition/master/custom_components/owlintuition/manifest.json",
+  "https://raw.githubusercontent.com/custom-components/sensor.owlintuition/master/custom_components/owlintuition/sample.owl2.xml",
```

Comment:
I have included this file as a form of documentation, but it is not needed to run the component. Thus I'd remove it from here.